tests/sles4sap/saptune/mr_test.pm

```diff
@@ -55,6 +55,8 @@ sub setup {
     if (check_var('BACKEND', 'qemu')) {
         # Ignore disk_elevator on VM's
         assert_script_run "sed -ri '/:scripts\\/disk_elevator/s/^/#/' \$(fgrep -rl :scripts/disk_elevator Pattern/)";
+        # Skip nr_requests on VM's
```

Comment:
Can you just add a reference to bsc#1177888, something like `see bsc#1177888 for more details`?

Reply:
> Can you just add a reference to bsc#1177888, something like `see bsc#1177888 for more details`?

Done!